### PR TITLE
Match response family spec

### PR DESCRIPTION
**Pull request by ellisjoe**

### CODE REVIEW

**Review by markelliot — APPROVED** [review]

File: okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/metrics/HostMetrics.java [hunk 1]
```diff
@@ -40,11 +40,11 @@
 
     /** Creates a metrics registry for calls from the given service to the given host. */
     public HostMetrics(TaggedMetricRegistry registry, String serviceName, String hostname) {
-        informational = registry.meter(name(serviceName, hostname, "informational"));
-        successful = registry.meter(name(serviceName, hostname, "successful"));
-        redirection = registry.meter(name(serviceName, hostname, "redirection"));
-        clientError = registry.meter(name(serviceName, hostname, "client-error"));
-        serverError = registry.meter(name(serviceName, hostname, "server-error"));
+        informational = registry.meter(name(serviceName, hostname, "1xx"));
```
Comment (markelliot) [inline okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/metrics/HostMetrics.java]: also these *xx things should be constants.

Comment (ellisjoe) [reply okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/metrics/HostMetrics.java]: done. left them as package private so a couple tests couldn't pick them up. will update once there's a public `MetricTags` in tritium

Comment (ellisjoe) [reply okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/metrics/HostMetrics.java]: https://github.com/palantir/tritium/pull/60

Comment (schlosna) [reply okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/metrics/HostMetrics.java]: Tritium 0.8.5

**Review by uschi2000 — APPROVED** [review]

**Pull request merged**